jupyter_client/tests/test_multikernelmanager.py

```diff
@@ -206,10 +200,10 @@ def test_subclass_callables(self):
         km.get_kernel(kid).reset_counts()
         km.reset_counts()
         km.shutdown_all(now=True)
-        assert km.call_count('shutdown_kernel') == 0
+        #assert km.call_count('shutdown_kernel') == 0
```

Comment:
Since we're now _async-first_, I think going with the expected sequence in the async test is the correct approach.  This does mean that the framework no longer calls `request_shutdown` and `finish_shutdown` on {A}MKM classes, so we might want to discuss their deprecation.